components/pkg-cfize/bin/hab-pkg-cfize.sh

```diff
@@ -75,7 +75,7 @@ exit_with() {
   exit $2
 }
 
-dockerize_tag() {
+dockerize_tags() {
```

Comment:
`$docker_output_file` has three tags by default. For example:
```
jbauman/sample-node-app:1.0.1-20180402210339
jbauman/sample-node-app:1.0.1
jbauman/sample-node-app:latest
```
I changed the cfize code to handle all 3 but just use the first one in the docker `FROM` command. Alternatively, we could use only the first with `grep --max-count=1` and make fewer changes, but this seemed more correct.

Reply:
This _seems_ reasonable to me!